packages/text/README.md

```diff
@@ -0,0 +1,38 @@
+# Collections
+
+[![Travis CI Status](https://img.shields.io/travis/monumentjs/core/master.svg?logo=travis)](https://travis-ci.org/monumentjs/core)
+[![Slack Chat](https://img.shields.io/badge/slack-chat-brightgreen.svg?logo=slack)](https://join.slack.com/t/monumentjs/shared_invite/enQtNDY1ODA1MTExMzQ4LTI0MjllODEwOTk5MjM0NGIwY2YwNzVjNDU3YjEwYzYwYTNjMmI0NjFkNmNjMDFlMjA1NzgzODk0NjcxZTc4NjM)
+[![npm version](https://badge.fury.io/js/%40monument%2Ftext.svg)](https://badge.fury.io/js/%40monument%2Ftext)
+![npm version](https://david-dm.org/monumentjs/text.svg)
+
+Provides text manipulation solutions
+
+Related links:
+
+- [API reference](https://monumentjs.github.io/package/collections)
```

Comment:
link should point to `text` package

Reply:
Good catch - I was sure I changed it:+1:
Fixed to point to text